source/includes/steps-create-index.yaml

```diff
@@ -27,18 +27,55 @@ content: |
 
    |compass-short| supports the following index options:
 
-   * `Build the index in the background <https://docs.mongodb.com/master/core/index-creation/index.html#background-construction>`_
+   .. list-table::
+      :header-rows: 1
+      :widths: 40 60 20
 
-   * :manual:`Create a unique index </core/index-unique>`
+      * - Option
+        - Description
+        - More Information
 
-   * :manual:`Create a TTL Index </core/index-ttl>`
+      * - Build index in the background
 
-   * :manual:`Create a partial index </core/index-partial/>`
+        - If checked, the MongoDB deployment remains available during
+          the index build operation.
 
-   * `Use custom collation <https://docs.mongodb.com/manual/indexes/index.html#indexes-and-collation>`_
+        - :manual:`Background Construction </core/index-creation/index.html#background-construction>`
 
-     For information on the Custom Collation settings, see :manual:`Collation Document </reference/collation/#collation-document>`
-     in the MongoDB manual.
+      * - Create unique index
+
+        - If checked, ensure that the indexed fields do not
+          store duplicate values.
+
+        - :manual:`Unique Indexes </core/index-unique>`
+
+      * - Create :abbr:`TTL (Time to Live)`
+
+        - If checked, automatically delete documents after a
+          specified number of seconds since the indexed field value.
+
+        - :manual:`TTL Indexes </core/index-ttl>`
+
+      * - Partial filter expression
+      
+        - If checked, only index documents which match the specified
+          filter expression.
+      
+        - :manual:`Partial Indexes </core/index-partial/>`
+
+      * - Use custom collation
+
+        - If checked, create a custom collation for the index
+          using the options provided in |compass-short|.
+
+        - :manual:`Collation Document </reference/collation/#collation-document>`
+
+      * - Wildcard projection (*New in MongoDB 4.2*)
```

Comment:
See above about parallelism. Possibly: "Use wildcard projection".

Nit: I wonder if we could put a line break between "Wildcard projection" and "(New in MongoDB 4.2.)"?

Reply:
1. ibid my other comment about these names matching what is shown in the UI, but will discuss with product.

2. 
<img width="1446" alt="Screen Shot 2019-07-24 at 4 37 36 PM" src="https://user-images.githubusercontent.com/29379027/61826909-6d2ff200-ae31-11e9-8535-ef5b7de90226.png"> unfortunately it looks like this is the only way we can get it to show up with a line break, which looks a bit weird imo. I think it's not ideal but having it on the same line looks a bit better. What do you think?

Reply:
Ah, ok. I agree to keep it on the same line. Thanks for trying!